Loenn/entities/alphalerplightsources.lua

```diff
@@ -0,0 +1,89 @@
+local contorthelper = require("mods").requireFromPlugin("libraries.contorthelper")
+
+local light = {}
+
+light.name = "ContortHelper/AlphaLerpLightSource"
+light.depth = 0
+light.texture = "ahorn/muntheory/contorthelper/alphalerplightsource"
+light.placements = {
+    name = "default",
+    data = {
+        color = "White",
+        radius = 48.0,
+        startFade = 24,
+        endFade = 48,
+        alphaA = 0.5,
+        alphaB = 1.0,
+        alphaEaser = "CubeInOut",
+        alphaSpeed = 1.0,
+        inverted = "false"

```

Comment:
`ìnverted` is set the `"false"` by default, but its possible values are `"False"`, `"True"` or `"Random"`. No idea is Lönn enforces that casing matches, but just to be sure, I'd change it.
```suggestion
        inverted = "False"
```